packages/ra-core/src/controller/button/useDeleteWithUndoController.spec.tsx

```diff
@@ -47,4 +48,54 @@ describe('useDeleteWithUndoController', () => {
             timeout: 1000,
         });
     });
+
+    it('should display success message after successful deletion', async () => {
+        const successMessage = 'Test Message';
+        const dataProvider = testDataProvider({
+            delete: jest.fn().mockResolvedValue({ data: {} }),
+        });
+
+        const MockComponent = () => {
+            const { handleDelete } = useDeleteWithUndoController({
+                record: { id: 1 },
+                resource: 'posts',
+                successMessage,
+            } as UseDeleteWithConfirmControllerParams);
+            return <button onClick={handleDelete}>Delete</button>;
+        };
+
+        let notificationsSpy;
+        const Notification = () => {
+            const { notifications } = useNotificationContext();
+            React.useEffect(() => {
+                notificationsSpy = notifications;
+            }, [notifications]);
+            return null;
+        };
+
+        render(
+            <TestMemoryRouter>
+                <CoreAdminContext dataProvider={dataProvider}>
+                    <MockComponent />
+                    <Notification />
+                </CoreAdminContext>
+            </TestMemoryRouter>
+        );
+
+        const button = screen.getByText('Delete');
+        fireEvent.click(button);
+
+        await waitFor(() => {
+            expect(notificationsSpy).toEqual([
+                {
+                    message: successMessage,
+                    type: 'info',
+                    notificationOptions: {
+                        messageArgs: { smart_count: 1 },
+                        undoable: false,
```

Comment:
I was surprised this test was passing with `undoable: false` while I would have expected `undoable: true` since we are in undoable mode.

Then I realized there is a (pre-existing) bug in this file: we are not importing `useDeleteWithUndoController` from the right place!

```diff
 import useDeleteWithUndoController, {
-    UseDeleteWithConfirmControllerParams,
-} from './useDeleteWithConfirmController';
+    UseDeleteWithUndoControllerParams,
+} from './useDeleteWithUndoController';
```

Can you fix this import (and also the test) please? :pray: 

```suggestion
                        undoable: true,
```

Reply:
I fixed it, thanks for the review.